pages/common/test.md

```diff
@@ -20,6 +20,6 @@
 
 `test ! -d "{{path/to/directory}}"`
 
-- If A is true, then do B, or C in the case of an error (notice that C may run even if A fails):
+- If condition is true, echo "true" runs, if the condition is false then echo "false" runs:
```

Comment:
```suggestion
- Run first command if a specific condition is true, otherwise a second one:
```

Optional fix: sentence generalization: sentence can be generalized not to update it later on placeholder content change.